packages/MJCore/src/generic/entityInfo.ts

```diff
@@ -1160,10 +1197,10 @@ export type ValidationErrorType = typeof ValidationErrorType[keyof typeof Valida
 export class ValidationErrorInfo {
     Source: string
     Message: string
-    Value: string
+    Value: any
```

Comment:
A little more type-safe alternative would be to use unknown

```suggestion
    Value: unknown
```

It'll accept any input,  just like `any` but will help to ensure some kind of type checking/casting is done before actually using it (unless passlng to another `unknown` param).